src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,84 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  fillteredTodos: Todo[];
+  setHasTodoClick: (a: boolean) => void;
+  hasTodoClick: boolean;
+  setUserId: (a: number) => void;
+  setTodoId: (a: number) => void;
+  todoId: number;
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({
+  fillteredTodos,
```

Comment:
The prop 'fillteredTodos' is misspelled. It should be 'filteredTodos'. Ensure consistency with the corrected variable name.